NPCAP.md

```diff
@@ -5,8 +5,8 @@ If you'd like to bump the npcap version please follow the below steps:
 1) Update `NPCAP_VERSION` value in the `Makefile`.
   * **NOTE**: Make sure the PR adding this is back-ported to the Go versions required by the Packetbeat CrossBuild target in [the mage file](https://github.com/elastic/beats/blob/main/x-pack/packetbeat/magefile.go). This is specified in the beats `.go-version` file.
 2) Download the new artifact.
-3) Upload the artifact to `gs://ingest-buildkite-ci/private`.
-  * **NOTE**: This particular Google Bucket can be accessible only by Elasticians who have got access to the Google project called `elastic-platform-ingest`.
+3) Upload the artifact to `gs://golang-crossbuild-ci-internal/private`.
```

Comment:
I think that we also need to adjust the beats repo (https://github.com/elastic/beats/blob/67636970b4225582683cca7541c27163452e59d3/x-pack/packetbeat/scripts/mage/pcap.go#L92) shortly after if we change buckets, otherwise the artifacts there e.g. `npcap` will get out of sync overtime.

Reply:
See https://github.com/elastic/beats/pull/44259